recipes/platform.delegates/all/conanfile.py

```diff
@@ -0,0 +1,70 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+
+class PlatformInterfacesConan(ConanFile):
+    name = "platform.delegates"
+    license = "MIT"
+    homepage = "https://github.com/linksplatform/Delegates"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = """platform.delegates is one of the libraries of the LinksPlatform modular framework, which uses 
+    innovations from the C++17 standard, for easier use delegates/events in csharp style."""
+    topics = ("linksplatform", "cpp17", "delegates", "events", "header-only")
+    settings = "os", "compiler", "build_type", "arch"
```

Comment:
Only compiler looks used

```suggestion
    settings = "compiler"
```